doc/conf.py

```diff
@@ -121,7 +121,7 @@
 
 redirects = {
     'README.md': '/',
-    'netplan': '/netplan-yaml',
+    'netplan': '/reference/netplan-yaml',
```

Comment:
todo: I'm still missing a redirect for the old `netplan-yaml` here.

In the end, we need to be able to redirect something like https://netplan.io/reference#routing to https://netplan.readthedocs.io/en/latest/reference/netplan-yaml/#routing

I was unable to make the `sphinx-reredirects` redirect any anchor (i.e. the URI fragment after the hashtag `#routing`), but this is important for the `netplan-yaml` page, as there are lots of deep links aroud the internet pointing to specific configuration and we would break all of them if we don't support redirecting to the proper anchor.

@rkratky Do you have any idea how anchor redirects could be solved?

Reply:
When you say you're missing a redirect for the old `netplan-yaml`, what do you mean @slyon?